### PR TITLE
Add missing zdbId

### PR DESCRIPTION
**Pull request by TobiasNx**

### CODE REVIEW

**Review by blackwinter — APPROVED** [review]
I suppose it's okay, so :+1:

(For future reference, `035$$a` with prefix `(DE-599)` is the [EKI](https://verbundwiki.gbv.de/display/VZG/Erstkatalogisierungs-ID).)

But while you're at it, you might also want to map `016$7#$a` with `$$2=DE-600`.

**Comment by TobiasNx** [comment]
Added the requested transformation which indeed catched a missing zdbId. I also added an dnbId and an object for dnb resources since they are provided in 016 too.

**Comment by TobiasNx** [comment]
@acka47 is it okay to map the dnbId of the record too?

File: src/test/resources/alma-fix/990103899140206441.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "almaMmsId" : "990103899140206441",
   "oclcNumber" : [ "1367753905" ],
   "zdbId" : "590016-5",
+  "dnbId" : "01335941X",
```
Comment (dr0i) [inline src/test/resources/alma-fix/990103899140206441.json:6]: This field `dnbId` has no entry in the `context.jsonld`.
Can you provide one ?

Comment (TobiasNx) [reply src/test/resources/alma-fix/990103899140206441.json:6]: I added it now

**Review by dr0i — APPROVED** [review]

**Comment by TobiasNx** [comment]
@acka47 approved this

**Comment by dr0i** [comment]
Will be deployed next Monday.

**Pull request merged**